### PR TITLE
refactor(organisms): SfContentPages

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue
https://github.com/DivanteLtd/storefront-ui/issues/806 

# Scope of work
refactor SfContentPages
need to be verified after SfTabs will be ready

# Screenshots of visual changes
mobile:
image
image

desktop:
image

image

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))
 - [ ] I accept [Individual Contributor License Agreement](https://github.com/DivanteLtd/next/blob/master/CLA.md)

### CODE REVIEW

File: packages/shared/styles/components/organisms/SfContentPages.scss [hunk 1]
```diff
@@ -1,95 +1,85 @@
 @import "../../helpers";
 
 .sf-content-pages {
+  height: calc(100vh - 3.125rem);
   overflow: hidden;
-  height: 100vh;
+  .sf-bar {
+    position: var(--content-pages-bar-position, fixed);
+    --bar-height: 3.125rem;
```
Comment (aniamusial) [inline packages/shared/styles/components/organisms/SfContentPages.scss]:
```suggestion
    --bar-height: 3.125rem;
    position: var(--content-pages-bar-position, fixed);
```

Can we change the modifier withoust additional `.sf-bar` class? :)

File: packages/shared/styles/components/organisms/SfContentPages.scss [hunk 2]
```diff
@@ -1,95 +1,85 @@
...
-      --content-pages-title-font,
-      font(
-        --content-pages-title,
-        500,
-        var(--font-size-big),
-        1.6,
-        var(--font-family-secondary)
-      )
+    margin: var(--content-pages-sidebar-title-margin, 0 0 0 var(--spacer-base));
+    @include font(
```
Comment (aniamusial) [inline packages/shared/styles/components/organisms/SfContentPages.scss]: Can you please replace all the `<h1> <h2>` with `SfHeading ` with a proper `level` prop

File: packages/shared/styles/components/organisms/SfContentPages.scss [hunk 3]
```diff
@@ -1,95 +1,85 @@
...
       --content-pages-category-title-font,
-      font(
-        --content-pages-category-title,
-        500,
-        var(--font-size-big),
-        1.6,
-        var(--font-family-secondary)
-      )
+      var(--font-normal),
```
Comment (aniamusial) [inline packages/shared/styles/components/organisms/SfContentPages.scss]: just like in comment above - use SfHeading, so we wont have to set fonts :)

File: packages/shared/styles/components/organisms/SfContentPages.scss (hunk 1 above)
Comment (pspaczek) [reply packages/shared/styles/components/organisms/SfContentPages.scss]: we shouldn't adjusted included component, it should be ok out of the box :)

File: packages/shared/styles/components/organisms/SfContentPages.scss (hunk 3 above)
Comment (pspaczek) [reply packages/shared/styles/components/organisms/SfContentPages.scss]: nope. we should use html tags here :)

**Pull request merged**